composer.json

```diff
@@ -4,7 +4,7 @@
 		"optimize-autoloader": true,
 		"classmap-authoritative": true,
 		"platform": {
-			"php": "8.0"
+			"php": "8.1"
```

Comment:
otherwise `nextcloud/ocp` stays at some very old version due to other dependencies that could not be updated.
Meaning this is required to update a version that known `addInitScript`...